release/build_release.sh

```diff
@@ -139,6 +154,8 @@ function main {
 
 		lc_time_run upload_release
 	else
+		#lc_time_run create_branch
```

Comment:
@kiwm, please create a dedicated commit just for this line to make clear to the people why we're temporarily disabling this line. Besides, create this new commit after the commit which creates the function `create_branch`.